frontend/admin/src/js/components/apiManagedTable/basicTableHelpers.tsx

```diff
@@ -1,5 +1,14 @@
 import { InputMaybe, OrderByClause, SortOrder } from "@common/api/generated";
 import React from "react";
+import { IntlShape } from "react-intl";
+
+import CheckButton from "@common/components/CheckButton";
+import { Scalars } from "../../api/generated";
+import { IndeterminateCheckbox } from "../Table/tableComponents";
```

Comment:
'IndeterminateCheckbox' is defined but never used.

Reply:
Fixed with [remove unused import](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3279/commits/d75cd62230c3c0fdd05f132dc876538660ed9114)